src/index.js

```diff
@@ -2,18 +2,18 @@ import { domElement } from './domElements.js'
 import { topSection } from './topSection.js'
 import { bottomSection } from './bottomSection.js'
 import { middleSection } from './middleSection.js'
-import * as topProto from './basePrototype.js'
-// import * as topClass from './baseClass.js'
+// import * as topProto from './basePrototype.js'
```

Comment:
Let's remove the unneeded commented lines

Reply:
sure I will resolve it in next pr